### DISCUSSION TITLE
【Q】 Do I need to modify this library to use it on my Japanese NTSC-J standard TV?

### DISCUSSION THREAD

**Discussion by riraosan** [Roger-random/ESP_8_BIT_composite#17] [Q&A]
Hello. @Roger-random
I am grateful to be able to use your library. Thank you.👍

I wrote your library into lolin32 and tweaked it into the composite input of a Japanese SHARP digital TV.
When I did so, I was able to display the text and graphics, but the text and graphics oscillated up and down periodically. I am not sure, but I am wondering if there is something wrong with the vertical sync signal.
Is this library not compatible with "NTSC-J"?
How can I correct the vertical oscillation of the display?

Thank you very much for your help.

PS
Maybe it's just noise coming from the ESP32 board.

**Comment by riraosan** [comment]
Hi. @Roger-random 

I found something in video_sync() that is different from the ESP_8_bit project.
I don't understand it very well, so this may be a strange question, but I'll ask it.

Why is there a difference between the [video_sync()](https://github.com/riraosan/ESP_8_BIT_composite/blob/a232df2517d60aa07a5fa1491914c91336c8e6d8/ESP_8_BIT_composite.cpp#L571) code in your library and the ESP_8_bit project's [video_sync()](https://github.com/rossumur/esp_8_bit/blob/c65fa1e70e3a981abb9f232ef481fcf0a42eab0a/src/video_out.h#L769)?

**Reply by riraosan** [reply]
That was a dumb question. Please ignore it.

**Comment by Roger-random** [comment]
As stated in the README, all of the technical underpinnings of video signal generation came from ESP_8_BIT.

I have no knowledge of NTSC signal generation and I do not know how it differs from NTSC-J.

If anyone manages to figure it out, we can talk about incorporating results of their discovery into the library.

**Reply by riraosan** [reply]
Hi @Roger-random 
I'll ask the author of the ESP_8_BIT library.
Thanks, 👍
